packages/design-system/package.json

```diff
@@ -0,0 +1,159 @@
+{
+  "name": "@livechat/design-system",
+  "version": "0.5.6",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "LiveChat Design System React components with css styles",
+  "homepage": "https://livechat.github.io/design-system/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/livechat/design-system"
+  },
+  "typings": "src/interfaces/index.d.ts",
+  "main": "dist/design-system.cjs.js",
+  "module": "dist/design-system.es.js",
+  "jsnext:main": "dist/design-system.es.js",
+  "style": "dist/design-system.css",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "scripts": {
+    "start": "rollup -c --watch",
+    "build": "rollup -c",
+    "prebuild": "rimraf dist",
+    "build:cdn": "npm run build && mkdir -p dist/cdn && cp dist/design-system.css dist/cdn/styles.css && cp dist/cdn/styles.css dist/cdn/styles-v$npm_package_version.css",
+    "prepare": "npm run build",
+    "test": "jest",
+    "test:update": "jest -u",
+    "test:watch": "jest --watchAll",
+    "test:coverage": "jest --coverage",
+    "eslint": "eslint --ext .js,.ts src/ ",
+    "eslint:fix": "eslint --ext .js,.ts src/ --fix "
+  },
+  "husky": {
```

Comment:
as mentioned elsewhere - this config should be moved to the root directory

Reply:
moved